src/main/Controls.js

```diff
@@ -86,6 +94,18 @@ class Controls extends React.Component<Props> {
     this.zoomByFactor(2.0);
   }
 
+  zoomAbs(level: number) {
```

Comment:
Can you comment what this function does?

Reply:
Done.